### ISSUE TITLE
Solution to power from 12V

### ISSUE THREAD

**Issue by reneklootwijk** [IOT-MCU/ESP-12S-A9-A9G-GPRS-Node-v1.0#22]
Does anybody have a good solution to power this module from 12V (car)?

**Comment by ayazurrehman786** [comment]
Yes, you can use 7805 voltage regulator to power up the module from 12V

**Comment by reneklootwijk** [comment]
Where should I hook this up when I want to use the battery as a failback? Via the micro USB connector?

**Comment by ayazurrehman786** [comment]
Connect it before J1 (USB port) . Output of 7805 must be connected to USB port it will charge the lithium battery also

**Comment by leifnel** [comment]
A better solution would be to take a lighter socket to usb charger adapter. It's a switching mode (i.e. not wasting heat as a 7805 linear regulator, and it's designed to take the variable and noisy voltage in a car environment,

You could take the electronics from the adapter and put it into the box with the board.
